### PR TITLE
ENH: added T1 Map as an optional input

### PR DESCRIPTION
**Pull request by stevedaxiao**
Once again. This is still a test. I probably made some mistakes to close the previous pull request.
Please let me know if it works now.

### CODE REVIEW

**Comment by fedorov** [comment]
The other changes related to renaming "mapping" to "map" are not in this pull request.

What do you mean by "this is still a test" ?

**Comment by stevedaxiao** [comment]
What do you mean "The other changes related to renaming "mapping" to "map""

**Comment by stevedaxiao** [comment]
I changed the " CLI/PkModeling.xml , CLI/PkModeling.cxx, the comments in CLI/itkSignalIntensityToConcentrationImageFilter.h and CLI/itkSignalIntensityToConcentrationImageFilter.cxx"
You will still see T1mapping somewhere. But this will never show up when users use it. 
I can change it , but it probably will take a little bit more time for testing and etc. Right now, it works.
let me know what I shall do, change or not?
"This is a test" means that in case it doesn't work.

Thanks

File: CLI/PkModeling.cxx [hunk 1]
```diff
@@ -298,6 +298,16 @@ int DoIt( int argc, char * argv[], const T1 &, const T2 &)
     aifMaskVolume = aifMaskVolumeReader->GetOutput();
     }
 
+  //Read T1 Map
+  typename MaskVolumeReaderType::Pointer T1MappingVolumeReader = MaskVolumeReaderType::New();
+  typename MaskVolumeType::Pointer T1MappingVolume = 0;
+  if (T1MapFileName != "")
+    {
+    T1MappingVolumeReader->SetFileName(T1MapFileName.c_str() );
```
Comment (fedorov) [inline CLI/PkModeling.cxx]: here

File: CLI/itkSignalIntensityToConcentrationImageFilter.hxx [hunk 2]
```diff
@@ -56,6 +56,13 @@ void SignalIntensityToConcentrationImageFilter<TInputImage, TMaskImage, TOutputI
     roiMaskVolumeIter.GoToBegin();
     }
 
+  InputMaskConstIterType T1MappingVolumeIter;
```
Comment (fedorov) [inline CLI/itkSignalIntensityToConcentrationImageFilter.hxx]: here

**Comment by fedorov** [comment]
> But this will never show up when users use it. 

You should think not only about users, but also about developers who will look at your code.

> "This is a test" means that in case it doesn't work.

I don't understand what you mean.

**Comment by stevedaxiao** [comment]
Let me know if it works right now. Github does not allow me squeeze this moment. I may do that after merge.

**Comment by stevedaxiao** [comment]
Sorry. Typo. Github doesn't recommend me squash commits at this moment.
Thanks.

**Comment by fedorov** [comment]
I don't understand.

**Comment by fedorov** [comment]
We can review what is going on with your github when we meet Wednesday.

**Pull request merged**